p2panda-auth/src/group/resolver.rs

```diff
@@ -25,13 +25,15 @@ use crate::traits::{GroupStore, IdentityHandle, Operation, OperationId, Ordering
 /// operations are processed. It can be assumed that the behavior is equivalent for an admin
 /// member being removed, or demoted from admin to a lower access level.
 ///
-/// 1) Removals
+/// ## Strong Remove Concurrency Rules
```

Comment:
Struggling a bit to call it "strong" with the rules around re-adding (see https://github.com/p2panda/p2panda/issues/760), but that maybe gets resolved in the future

Reply:
If feel feel strongly we should rename this to just "Concurrency Rules" and not refer to "Strong" I wouldn't mind doing that. 

My position right now is that we made an attempt to arrive at "Strong Remove" logic, but there is a question around how we handle re-adds which needs resolving. We should find a solution we're happy with which we feel provides the strong remove property we're after. For that reason (that we will make changes until we're happy) I'm also ok with leaving the description as it is (keeping the "Strong").

Reply:
My worry is that "strong removal" might indicate a false security with "full guarantee around removals in a byzantine setting" while it doesn't fully provide that. I understand that almost all other guarantees are given, but for a more "activist context" it's still not enough.

Reply:
I still need to consider the current approach (re-adds respected) as an attack vector, not saying it isn't, but I haven't fully understood it yet.

Reply:
> I still need to consider the current approach (re-adds by valid members respected) as an attack vector, not saying it isn't, but I haven't fully understood it yet.

Yes, no rush here! I think the issue consists of two questions: 1. Attacks 2. Intuition / expectations around what it means to "remove" someone from a group (especially when removals are always "strong")

Reply:
Cool, I suggest we keep the name and explore the open questions to assure we fulfil the expectations of that name.